dist/common/scripts/scylla_io_setup

```diff
@@ -149,6 +149,7 @@ if __name__ == "__main__":
                 disk_properties = {}
                 disk_properties["mountpoint"] = datadir()
                 nr_disks = idata.nvmeDiskCount
+                # below is based on https://cloud.google.com/compute/docs/disks/local-ssd#performance
                 if nr_disks >= 1 & nr_disks < 4:
```

Comment:
Please fold fixes into the patches that introduce the problems. I'm not doing this folding in my mind.

Reply:
so there are patches based on your comments, e.g.
https://github.com/scylladb/scylla/pull/6632/commits/3f5a8b0a6ed1b4e4e843d4c33c454b1867ffaec6
(see small selector on left top side of patch view, you can select which patches you want to see there, even create sets of them to review what changed since you last reviewed)
I hope this is what you meant, or you want me to squash the patches?
(or split current patches to smaller ones?)

Reply:
Each patch should be about a single topic.

Changes that fix review comments should be folded into the patches that had the problems in the first place.